xcube_server/__init__.py

```diff
@@ -20,5 +20,5 @@
 # SOFTWARE.
 
 # Version string (PEP440-compatible), e.g. "0.8.0", "0.8.0.dev1", "0.8.0rc1", "0.8.0rc1.dev1"
-__version__ = '0.1.0.dev4'
+__version__ = '0.1.0.dev5'
```

Comment:
If you update the version, you should do it everywhere - also in the readme, in PKG-INFO, and the other places.